expat/tests/xmltest.sh

```diff
@@ -50,12 +50,13 @@ RunXmlwfNotWF() {
   fi 
 }
 
-# RunXmlwfWF file reldir
+# RunXmlwfWF file reldir [additional options]
 # reldir includes trailing slash
 RunXmlwfWF() {
   file="$1"
   reldir="$2"
-  $XMLWF -p -N -d "$OUTPUT$reldir" "$file" > outfile || return $?
+  options="$3"
```

Comment:
Since options is plural, let's make it support multiple options. How about:
```
file="$1"
shift
reldir="$1"
shift
options=( "$@" )
$XMLWF -p -N -d "$OUTPUT$reldir" "${options[@]}" "$file" > outfile || return $?
```
Would that work?